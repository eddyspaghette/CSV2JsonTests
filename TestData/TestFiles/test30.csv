ッ!
ツ!ぃ!ぎ!せ!℟!▓
ヅ!い!け!₠!Å!○
ミ!ぅ!げ!₡!ℬ!☝
デ!う!こ!₢!Δ!⭐
ト!ぇ!ご!₣!ψ!😵
ド!え!さ!℀!ↂ!🍕
ツ!ぃ!ぎ!せ!℟!▓
デ!う!こ!₢!Δ!⭐
ト!ぇ!ご!₣!ψ!😵
