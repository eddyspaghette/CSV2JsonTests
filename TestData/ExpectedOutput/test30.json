[
 {
   "ッ!": "ツ!ぃ!ぎ!せ!℟!▓"
 },
 {
   "ッ!": "ヅ!い!け!₠!Å!○"
 },
 {
   "ッ!": "ミ!ぅ!げ!₡!ℬ!☝"
 },
 {
   "ッ!": "デ!う!こ!₢!Δ!⭐"
 },
 {
   "ッ!": "ト!ぇ!ご!₣!ψ!😵"
 },
 {
   "ッ!": "ド!え!さ!℀!ↂ!🍕"
 },
 {
   "ッ!": "ツ!ぃ!ぎ!せ!℟!▓"
 },
 {
   "ッ!": "デ!う!こ!₢!Δ!⭐"
 },
 {
   "ッ!": "ト!ぇ!ご!₣!ψ!😵"
 }
]